js/mail.js

```diff
@@ -675,6 +683,16 @@ var Mail = {
 					});
 					// Remove spinner when loading finished
 					$('iframe').parent().removeClass('icon-loading');
+					// Add body content to inline reply (html mails)
+					var text = $(this).contents().find('body')[0].innerText;
```

Comment:
I get an error in FF saying ``TypeError: text is undefined``

I fixed it by using ``var text = $(this).contents().find('body').first().text();``